core/os/os2/errors_linux.odin

```diff
@@ -1,145 +1,164 @@
-ENOKEY         :: 126  /* Required key not available */
-EKEYEXPIRED    :: 127  /* Key has expired */
-EKEYREVOKED    :: 128  /* Key has been revoked */
-EKEYREJECTED   :: 129  /* Key was rejected by service */
-EOWNERDEAD     :: 130  /* Owner died */
-ENOTRECOVERABLE:: 131  /* State not recoverable */
-ERFKILL        :: 132  /* Operation not possible due to RF-kill */
-EHWPOISON      :: 133  /* Memory page has hardware error */
-
-_get_platform_error :: proc(res: int) -> Error {
-	errno := unix.get_errno(res)
-	return Platform_Error(i32(errno))
+_errno_strings : [int(max(linux.Errno)) + 1]string = {
+	linux.Errno.NONE            = "Success",
+	linux.Errno.EPERM           = "Operation not permitted",
+	linux.Errno.ENOENT          = "No such file or directory",
+	linux.Errno.ESRCH           = "No such process",
+	linux.Errno.EINTR           = "Interrupted system call",
+	linux.Errno.EIO             = "Input/output error",
+	linux.Errno.ENXIO           = "No such device or address",
+	linux.Errno.E2BIG           = "Argument list too long",
+	linux.Errno.ENOEXEC         = "Exec format error",
+	linux.Errno.EBADF           = "Bad file descriptor",
+	linux.Errno.ECHILD          = "No child processes",
+	linux.Errno.EAGAIN          = "Resource temporarily unavailable",
+	linux.Errno.ENOMEM          = "Cannot allocate memory",
+	linux.Errno.EACCES          = "Permission denied",
+	linux.Errno.EFAULT          = "Bad address",
+	linux.Errno.ENOTBLK         = "Block device required",
+	linux.Errno.EBUSY           = "Device or resource busy",
+	linux.Errno.EEXIST          = "File exists",
+	linux.Errno.EXDEV           = "Invalid cross-device link",
+	linux.Errno.ENODEV          = "No such device",
+	linux.Errno.ENOTDIR         = "Not a directory",
+	linux.Errno.EISDIR          = "Is a directory",
+	linux.Errno.EINVAL          = "Invalid argument",
+	linux.Errno.ENFILE          = "Too many open files in system",
+	linux.Errno.EMFILE          = "Too many open files",
+	linux.Errno.ENOTTY          = "Inappropriate ioctl for device",
+	linux.Errno.ETXTBSY         = "Text file busy",
+	linux.Errno.EFBIG           = "File too large",
+	linux.Errno.ENOSPC          = "No space left on device",
+	linux.Errno.ESPIPE          = "Illegal seek",
+	linux.Errno.EROFS           = "Read-only file system",
+	linux.Errno.EMLINK          = "Too many links",
+	linux.Errno.EPIPE           = "Broken pipe",
+	linux.Errno.EDOM            = "Numerical argument out of domain",
+	linux.Errno.ERANGE          = "Numerical result out of range",
+	linux.Errno.EDEADLK         = "Resource deadlock avoided",
+	linux.Errno.ENAMETOOLONG    = "File name too long",
+	linux.Errno.ENOLCK          = "No locks available",
+	linux.Errno.ENOSYS          = "Function not implemented",
+	linux.Errno.ENOTEMPTY       = "Directory not empty",
+	linux.Errno.ELOOP           = "Too many levels of symbolic links",
+	41                          = "Unknown Error (41)",
```

Comment:
Maybe it's a good idea to add a pseudo-error in `sys/linux` instead for these missing gaps.